dbt/dbt_project.yml

```
@@ -11,9 +11,9 @@ profile: "pudl_dbt"
 # The `model-paths` config, for example, states that models in this project can be
 # found in the "models/" directory. You probably won't need to change these!
 model-paths: ["models"]
-macro-paths: ["macros"]
+macro-paths: ["macros", "tests/generic_tests"]
 seed-paths: ["seeds"]
-test-paths: ["tests"]
+test-paths: ["tests/singular_tests"]
```

Possibly blocking: the current schematic places singular tests in `tests/data_tests/generic_tests`

But also I think we still want dbt to be able to find the unit tests as well, which the schematic places in `tests/unit_tests`

So probably the original 
```suggestion
test-paths: ["tests"]
```
is superior?

---

Will it be an issue if the test path is `test` while the macro path is `tests/data_tests/generic_tests`? Will that overlap cause any problems? I.e.: the generic tests getting read twice?